junction/conferences/migrations/0001_initial.py

```diff
@@ -16,55 +16,184 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='Conference',
             fields=[
-                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
```

Comment:
Do we really need to format migrations as well ?

Reply:
🤷🏻‍♂️

I honestly don't care about it, and it's likely not going to matter much to my workflow (where I run the linters prior to a push).

Reply:
And it was easier to not have an exclude rule than to have one. None the less, it'll be easy enough to undo with a rebase, so lemme know what y'all think. :)

Reply:
Well given that we are using pre-commit which blackens the code before each commit, we don't have to invest in formatting it each time. So it seems good to go. But I see migrations still added to exclude section of flake8 pre-commit hook

Reply:
@pradyunsg Please have a look at the exclude section

Reply:
Could you clarify what you mean?